web-frontend/src/model/Item.ts

```diff
@@ -0,0 +1,4 @@
+export type Item = {
+    id: string
+    name: string
+}
```

Comment:
see above. Maybe better as an embedded type of subscriptions

Reply:
No, not here. Because one item can be in several boxes I will need that in the future.